step.yml

```diff
@@ -155,13 +161,25 @@ inputs:
           -derivedDataPath DERIVED_DATA_PATH`
 
         In case of `generate_code_coverage_files: "yes"`
-        `xcodebuild` becomes two additional flags:
+        `xcodebuild` gets two additional flags:
 
-          * GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES
-          * GCC_GENERATE_TEST_COVERAGE_FILES=YES
+        * GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES
+        * GCC_GENERATE_TEST_COVERAGE_FILES=YES
 
         If you want to add more options, list that separated by space character.
         Example: `-xcconfig PATH -verbose`
+  - single_build: "true"
+    opts:
+      title: "Run xcodebuild test only"
+      description: |-
+        If `single_build: false` step runs `xcodebuild OPTIONS build OPTIONS` befor the test,
+        to generate the project derived datas. After that comes `xcodebuild OPTIONS build test OPTIONS`,
+        which command's log is presented in the setp's log.
```

Comment:
If `single_build: false` step runs `xcodebuild OPTIONS build OPTIONS` befor the test,
 +        to generate the project derived datas. After that comes `xcodebuild OPTIONS build test OPTIONS`,
 +        which command's log is presented in the step's log.